lib/ts/Arena.h

```diff
@@ -21,12 +21,13 @@
   limitations under the License.
  */
 
-#ifndef __ARENA_H__
-#define __ARENA_H__
+#pragma once
```

Comment:
I am fine with the #pragma one

It is supported by all compiler I understand we support.
any gcc, icc and clang compiler supports this feature. the versions we care about it ( ie ones that support c++11) support this feature. I already had a PR go in with this same change